### DISCUSSION TITLE
No Scroll Bar for Sublist/Subtabs

### DISCUSSION THREAD

**Discussion by Spike0341** [burkybang/SuiteAdvanced-Feedback#116] [Q&A] [answered]
I searched but didn't see anything relating to this, so I apologize if I missed it.  I'm having an issue when I am browsing records. We have a probably too many subtabs on some of our records/sublists. When veiwing the records there is no scrollbar at the bottom so you can't see the rest of the tabs that spill off the page. I have to click on another subtab to "select" that element, then use the arrow keys to scroll left or right. When I do that it only scrolls the subtabs, which is fine. 

Has anyone else ran into this or found a fix? I can add media, but it will take me a bit to remove the PI from it. 

Thank you!

**Comment by burkybang** [comment, marked as answer]
Hi @Spike0341. If I understand correctly, this is an intended feature. It's to prevent having to scroll the entire page horizontally because of subtab overflow.

Please show screenshots though if there's something wacky going on with the UI.

There are three ways to scroll the subtabs.
1. Click a subtab that's near the edge in the direction you'd like to scroll. The selected subtab attempts to scroll to the middle of the screen automatically, so selecting a new one will force the subtab bar to scroll.
2. Hover your mouse on the left or right edge of the subtab bar. Hovering for a second causes the subtab bar to start scrolling.
3. Hover your mouse over any part of the sub tab bar, hold the shift key, and scroll your mouse wheel. This is works in any application to scroll horizontally - not just NetSuite with SuiteAdvanced.

**Reply by Spike0341** [reply]
Thanks for the quick response. I knew I was missing something. This is perfect.
